superset/views/core.py

```diff
@@ -171,6 +171,13 @@ class Superset(BaseSupersetView):  # pylint: disable=too-many-public-methods
 
     logger = logging.getLogger(__name__)
 
+    def __repr__(self):
```

Comment:
do you think it's safe for this to be guessable? Or should we encode it with the shared secret instead? 

I don't really know, but throwing it out there

Reply:
I can't think of a scenario where this will cause security issues. But I guess it does make it easier to invalidate cache if we encode it with some shared secret.